packages/connectivity_plus/connectivity_plus/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: connectivity_plus
 description: Flutter plugin for discovering the state of the network (WiFi & mobile/cellular) connectivity on Android and iOS.
-version: 2.3.8
+version: 2.3.9
```

Comment:
no need to bump the version of the main package when updating the macos one, can you revert this and remove the changelog entry?

Reply:
Sure, my bad. I got confused by 2nd paragraph from contributor.md.

> If updating a platform package (e.g. device_info_plus_macos), please remember to update version and changelog of the main package (e.g. device_info_plus).

Reply:
Done. Removed version bump from main package and removed changelog entry from main package.

Reply:
yeah, the federated plugin architecture is very confusing, and I'd love to simplify it someday. Thanks for taking care of this!